chat-plugins/info.js

```diff
@@ -406,7 +406,7 @@ const commands = {
 		target = sep[0].trim();
 		let targetId = toId(target);
 		if (!targetId) return this.parse('/help data');
-		let targetNum = parseInt(targetId);
+		let targetNum = parseInt(target);
 		if (!isNaN(targetNum) && '' + targetNum === target) {
```

Comment:
oh actually `'' + targetNum === target` protects against the thing I was afraid of

Reply:
What was it that you were afraid of?

Reply:
```
parseInt('10,000,000 volt thunderbolt')
-> 10
```

wanted to make sure `/dt 10,000,000 volt thunderbolt` didn't give you Caterpie

Reply:
This is really brittle, though, in that `/dt 10.` gives an error.

But I guess it was always that brittle.